hack/local-up-cluster.sh

```diff
@@ -509,23 +509,23 @@ EOF
 }
 
 # start_discovery relies on certificates created by start_apiserver
-function start_discovery {
-    # TODO generate serving certificates
+# function start_discovery {
```

Comment:
If the purpose is only to disable kubernetes-discovery, we do not need to remove this function.